management/rds_test.go

```diff
@@ -68,6 +68,8 @@ func TestAddRds(t *testing.T) {
 				TLSSkipVerify:             false,
 				DisableQueryExamples:      false,
 				TablestatsGroupTableLimit: 2000,
+				DisableBasicMetrics:       true,
```

Comment:
Let's add check that this fields are set correctly and add it for inventory API as well.